Build/Tasks/CreateUpgrade.cs

```diff
@@ -25,6 +25,7 @@ public sealed class CreateUpgrade : FrostingTask<Context>
         public override void Run(Context context)
         {
             this.RenameResourcesFor98xUpgrades(context);
+            this.RenameResourcesFor910xUpgrades(context);

```

Comment:
Considering that this code is only running for the current version releases, and is internal to the build process.

Could we combine the existing method and this method to something such as `RenameResourcesForUpgrades` which is more descriptive of the action necessary and leave the `[obsolete]` tag?

Reply:
Done

Reply:
I think @mitchelsellers was saying to leave the `Obsolete` attribute (we still want to remove this method in DNN 10), otherwise this looks good to me 👍🏻

Reply:
Yeah, I would also prefer to keep the obsolete so we don't forget to remove this.

Reply:
Done